### ISSUE TITLE
dynamic_rnn issue:

### ISSUE THREAD

**Issue by lipond** [tensorflow/tensorflow#6873]
when I run a copy of code based on tensorflow, it gives following error, what should I do next ?

Caused by op u'RNN/Assert/AssertGuard/Assert', defined at:
  File "lstm_and_ctc_ocr_train.py", line 188, in <module>
    train()
  File "lstm_and_ctc_ocr_train.py", line 78, in train
    logits, inputs, targets, seq_len, W, b = model.get_train_model()
  File "/home/sanjie/projects/tensorflow_lstm_ctc_ocr_ywh/model.py", line 111, in get_train_model
    outputs, _ = tf.nn.dynamic_rnn(cell, inputs, seq_len, dtype=tf.float32)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/rnn.py", line 823, in dynamic_rnn
    [_assert_has_shape(sequence_length, [batch_size])]):
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/rnn.py", line 818, in _assert_has_shape
    packed_shape, " but saw shape: ", x_shape])
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/control_flow_ops.py", line 140, in Assert
    condition, no_op, true_assert, name="AssertGuard")
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/control_flow_ops.py", line 1717, in cond
    _, res_f = context_f.BuildCondBranch(fn2)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/control_flow_ops.py", line 1613, in BuildCondBranch
    r = fn()
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/control_flow_ops.py", line 137, in true_assert
    condition, data, summarize, name="Assert")
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/gen_logging_ops.py", line 37, in _assert
    summarize=summarize, name=name)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/op_def_library.py", line 756, in apply_op
    op_def=op_def)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 2380, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 1298, in __init__
    self._traceback = _extract_stack()

InvalidArgumentError (see above for traceback): assertion failed: [Expected shape for Tensor sequence_length:0 is ] [64] [ but saw shape: ] [85]

**Comment by jart** [comment]
Thanks for reaching out. We try to keep this issue tracker limited to bugs and feature requests. We recommend posting on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) for community-driven support.
